draft-fossati-tls-attestation.md

```diff
@@ -444,15 +456,16 @@ binder_collection = {
 {: #figure-tls-binder title="Format of TLS Binder Collection."}
 
 * Nonce is the value provided as a challenge by the relying party.
-* The identity key public fingerprint (ik_pub_fingerprint) is a SHA256 hash of
-  the Subject Public Key Info from the leaf X.509 certificate transmitted in the
-  handshake.
+* The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
```

Comment:
the CDDL above implies a certain hash algorithm is to be used (i.e., SHA-256, SHA-512/256).  If we want this to be fixed we should say so.

A (maybe) better option is to use a self-descriptive digest data item, e.g. [COSE_Hash_Find](https://www.rfc-editor.org/rfc/rfc9054.html#section-2.1-6) so that we don't need to make hard choices for the implementations.

Yet another approach would be to treat it as opaque bytes and do not make the size fixed.

In all cases, I agree with the idea providing a "reasonable" upper-bound for the key id.

Reply:
> Yet another approach would be to treat it as opaque bytes and do not make the size fixed.

Yeah, I'd rather go down this route, I'd rather avoid any complexity that we need to heap onto the TLS implementation. It's mostly an oversight from a change prompted by Yaron earlier, to use whatever the TLS handshake negotiated instead of static SHA256.